src/App.js

```diff
@@ -1,21 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Quotes from './components/Quotes';
+import Home from './components/Home';
+import Footer from './components/Footer';
 import './App.css';
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <div className="main-container">
-        <Calculator />
-
-      </div>
-    );
-  }
-}
+const App = () => (
+  <div className="allmain">
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/calculator" element={<Calculator />} />
+      <Route path="/quotes" element={<Quotes />} />
```

Comment:
- [ ] I would suggest you add exact param because the exact param disables the partial matching for a route and makes sure that it only returns the route if the path is an EXACT match to the current URL